decidim-meetings/app/cells/decidim/meetings/meeting_card_metadata_cell.rb

```diff
@@ -6,7 +6,7 @@ module Meetings
     class MeetingCardMetadataCell < Decidim::CardMetadataCell
       alias meeting model
 
-      delegate :type_of_meeting, :start_time, :end_time, :category, :withdrawn?, to: :meeting
+      delegate :type_of_meeting, :start_time, :end_time, :taxonomies, :withdrawn?, to: :meeting
```

Comment:
The MeetingCardMetadataCell is actually using the taxonomy_items defined in CardMetadataCell. In that method, you are invoking resource.taxonomies.

```suggestion
      delegate :type_of_meeting, :start_time, :end_time, :withdrawn?, to: :meeting
```

Reply:
right, no need for this then